src/components/Footer.js

```diff
@@ -0,0 +1,56 @@
+import Link from 'next/link';
+import {useRouter} from 'next/router';
+import styled from 'styled-components';
+
+const StyledFooter = styled.footer`
+	position: fixed;
+	bottom: 0;
+	left: 0;
```

Comment:
As you have set already the `width` to 100%, you do not need this line.
```suggestion

```

Reply:
Ok, erledigt .